_posts/2024-05-23-pyos-pyconus-2024-recap.md

```diff
@@ -0,0 +1,258 @@
+---
+layout: single
+title: "pyOpenSci at PyCon US 2024 - Python packaging and community"
+excerpt: "Learn about  pyOpenSci's experience at PyCon US 2024, how pyOpenSci is making Python packaging more accessible and beginner friendly and how the Python community is helping us get there."
+author: "Leah Wasser"
+permalink: /blog/recap-pyos-pyconus-2024.html
+header:
+  overlay_image: images/blog/2024/may/pycon-us-2024.png
+  overlay_filter: 0.6
+categories:
+  - blog-post
+  - community
+classes: wide
+toc: true
+comments: true
+---
+
+<!-- TODOs:
+* embed video when live
+* add graphics from social that jesse sent
+* fix captions
+* add stravalib note at the end of my talk section -->
+
+
+<div class="notice" markdown = "1">
+## TL&DR
```

Comment:
I've usually seen this as either:
* TLDR
* TL;DR

Reply:
Fixed!